src/js/pages/services/ServicesTab.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import AlertPanel from '../../components/AlertPanel';
+import Config from '../../config/Config';
+import DCOSStore from '../../stores/DCOSStore';
+import FilterHeadline from '../../components/FilterHeadline';
+import SaveStateMixin from '../../mixins/SaveStateMixin';
+import Service from '../../structs/Service';
+import ServiceDetail from '../../components/ServiceDetail';
+import ServiceFilterTypes from '../../constants/ServiceFilterTypes';
+import ServiceSidebarFilters from '../../components/ServiceSidebarFilters';
+import ServicesTable from '../../components/ServicesTable';
+import ServiceTree from '../../structs/ServiceTree';
+import SidebarActions from '../../events/SidebarActions';
+import SidePanels from '../../components/SidePanels';
+
+var DEFAULT_FILTER_OPTIONS = {
+  filterHealth: null,
+  searchString: ''
+};
+
+let saveState_properties = Object.keys(DEFAULT_FILTER_OPTIONS);
+
+var ServicesTab = React.createClass({
```

Comment:
let's create this component by `class ServicesTab extends React.Component` (i forget the technical name for this)

i always refer to the component `ServiceOverlay` if i need a reference